src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/GetAzurePolicyAssignment.cs

```diff
@@ -18,60 +18,60 @@ namespace Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation
     using Microsoft.Azure.Commands.ResourceManager.Cmdlets.Extensions;
     using Microsoft.Azure.Commands.ResourceManager.Common;
     using Newtonsoft.Json.Linq;
+    using Policy;
     using System.Management.Automation;
     using System.Threading.Tasks;
 
     /// <summary>
     /// Gets the policy assignment.
     /// </summary>
-    [Cmdlet(VerbsCommon.Get, "AzureRmPolicyAssignment", DefaultParameterSetName = GetAzurePolicyAssignmentCmdlet.ParameterlessSet), OutputType(typeof(PSObject))]
+    [Cmdlet(VerbsCommon.Get, "AzureRmPolicyAssignment", DefaultParameterSetName = PolicyCmdletBase.DefaultParameterSet), OutputType(typeof(PsPolicyAssignment))]
```

Comment:
Did you read the design review comments here: https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/74? This change was rather insistently requested by @vladimir-shcherbakov, despite my initial objections. One concern I mentioned was that it could be a breaking change, although @vladimir-shcherbakov was confident it was fine. To be sure, I prototyped the change and did some testing. Upon study it appears that the only thing it is likely to break was a script using strict semantics and testing property existence to determine "present or not", rather than a check for $null. In non-strict mode both results are the same. Of course, since the underlying type is different, it's always possible to concoct an example it would break (e.g. explicit check for PSObject type), but those cases don't seem realistic to me. Am I missing other important breaking scenarios?

Reply:
>I might have missed it, but where are we returning a PsPolicyAssignment object rather than a PSObject now?

If your question is asking specifically where in the code the new object types are getting constructed, take a look here: https://github.com/mentat9/azure-powershell/blob/preview/src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Implementation/Policy/PolicyCmdletBase.cs in method GetOutputPolicyAssignments()

Reply:
Sorry, I should've been more explicit about the kind of breaking changes we look for in this case -- this change from `PSObject` to `PsPolicyAssignment` might not introduce a structural breaking change (one that would cause an existing script to throw) but could introduce a behavioral breaking change (script continues to work, but users don't get back the same result from one version to the next). My concern is that users have scripts that use properties from the object returned from `Get-AzureRmPolicyAssignment` that may not exist in the `PsPolicyAssignment` object, but we may need to do some more investigation to determine what values are returned from the server and if any of them don't show up as a property for `PsPolicyAssignment` (and for other types converted from `PSObject` in this PR). We can talk offline about this some more if needed since this is a change we would want to avoid making for the stable `AzureRM.Resources` module if it means sacrificing the piping scenario.

Reply:
The wrapper classes I created specifically include every property that comes back from the server. Let me know how to follow this up with your team.